fortuna/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Lol kind of ironic that the problem was that we couldn't communicate with the babel engine since our file was named the wrong thing...
Since Babel is named after a famous tower that didn't work because people couldn't communcate

Reply:
Lol ironic indeed.